### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by mdonnalley**
### What does this PR do?

Bumps our plugin versions

### What issues does this PR fix or reference?
[skip-validate-pr]

### CODE REVIEW

File: schemas/@sf/deploy-retrieve/deploy-metadata.json [hunk 1]
```diff
@@ -88,7 +91,7 @@
           "type": "number"
         },
         "time": {
-          "type": "number"
+          "type": "string"
```
Comment (RodEsp) [inline schemas/@sf/deploy-retrieve/deploy-metadata.json:94]: Why was this changed?
Seems like a breaking change.

**Review by RodEsp — APPROVED** [review]

**Review by peternhale — APPROVED** [review]
